android/app/src/main/java/org/electroncash/electroncash3/ColdLoad.kt

```diff
@@ -0,0 +1,105 @@
+package org.electroncash.electroncash3
+
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Intent
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.ViewModel
+import com.google.zxing.integration.android.IntentIntegrator
+import kotlinx.android.synthetic.main.load.*
+import kotlinx.android.synthetic.main.main.*
+
+// This provides a dialog to allow users to input a sting, which is then broadcast
+// on the bitcoin cash network. Strings are not validated, so invalid user inputs fail
+// gracefully and nothing happens. Valid transactoin quickly show up in transactions.
+
+class ColdLoadDialog : AlertDialogFragment() {
+
+    class Model : ViewModel() {}
+
+    //    val coldloadDialog by lazy { targetFragment as ColdLoadDialog }
```

Comment:
Leftover commented-out code should be removed.

Reply:
removed